contributing.json

```diff
@@ -0,0 +1,40 @@
+// https://gitmagic.io/rules
+{
+  "commit": {
+    "subject_cannot_be_empty": true,
+    "subject_must_be_longer_than": 4,
+    "subject_must_be_shorter_than": 101,
+    "subject_lines_must_be_shorter_than": 51,
+    "subject_must_be_single_line": true,
+    "subject_must_be_in_tense": "imperative",
+    "subject_must_start_with_case": "lower",
+    "subject_must_not_end_with_dot": true,
+
+    "body_lines_must_be_shorter_than": 73
+  },
+  "pull_request": {
+    "subject_cannot_be_empty": true,
+    "subject_must_be_longer_than": 4,
+    "subject_must_be_shorter_than": 101,
+    "subject_must_be_in_tense": "imperative",
+    "subject_must_start_with_case": "upper",
+    "subject_must_not_end_with_dot": true,
+
+    "body_cannot_be_empty": true,
+    "body_must_include_verification_steps": true,
+    "body_must_include_screenshot": ["html", "css"]
```

Comment:
That is a default one as it seems :D here we can see all rules: https://gitmagic.io/rules/